docs/content/docs/dev/table/materialized-table/statements.md

```diff
@@ -165,15 +165,15 @@ FRESHNESS = INTERVAL '5' HOUR
 ```sql
 -- The refresh mode of the created materialized table is CONTINUOUS, and the job's checkpoint interval is 1 hour.
 CREATE MATERIALIZED TABLE my_materialized_table
-    REFRESH_MODE = CONTINUOUS
     FRESHNESS = INTERVAL '1' HOUR
+    REFRESH_MODE = CONTINUOUS
     AS SELECT
        ...    
 
 -- The refresh mode of the created materialized table is FULL, and the job's schedule cycle is 10 minutes.
 CREATE MATERIALIZED TABLE my_materialized_table
-    REFRESH_MODE = FULL
     FRESHNESS = INTERVAL '10' MINUTE
+    REFRESH_MODE = FULL
```

Comment:
what is the purpose of this change - it seems to just move a line. I assume it works before.

Reply:
@davidradl I try to parse this sql which `REFRESH_MODE` before `FRESHNESS`and failed.
```shell
org.apache.calcite.sql.parser.SqlParseException: Encountered "REFRESH_MODE" at line 7, column 5.
Was expecting:
    "FRESHNESS" ...
```
So `FRESHNESS` must be before`REFRESH_MODE `.

Reply:
The original example is not correct, this follows the correct syntax.